container/features/src/main/resources/features.xml

```diff
@@ -98,11 +98,11 @@
       <bundle>wrap:mvn:org.apache.xmlgraphics/batik-xml/1.7</bundle>
     </feature>
 
-    <feature name="c3p0" description="c3p0" version="0.9.1.2">
+    <feature name="c3p0" description="c3p0" version="${c3p0Version}">
       <feature>postgresql</feature>
 
       <!-- Add all possible JDBC drivers to the c3p0 import list so that it can classload them -->
-      <bundle>wrap:mvn:c3p0/c3p0/0.9.1.2$Bundle-SymbolicName=c3p0&amp;Bundle-Version=0.9.1.2&amp;Import-Package=javax.management,javax.naming,javax.naming.spi,javax.sql,javax.xml.parsers,org.apache.log4j,org.postgresql,org.w3c.dom</bundle>
+      <bundle>wrap:mvn:com.mchange/c3p0/${c3p0Version}$Bundle-SymbolicName=c3p0&amp;Bundle-Version=${c3p0Version}&amp;Import-Package=javax.management,javax.naming,javax.naming.spi,javax.sql,javax.xml.parsers,org.apache.log4j,org.postgresql,org.w3c.dom</bundle>
```

Comment:
> On May 25, 2016, at 9:29 AM, Seth Leger <notifications@github.com> wrote:
> 
> Yes, exactly. The Karaf integation tests are only in develop, sorry for the confusion.

No worries.

> The c3p0 feature is not loaded on the OpenNMS side but it is loaded to satisfy dependencies inside Minion so we still need to test it to make sure it doesn't blow up. :)


Cool! I'm going to temporary apply the changes locally on develop to execute the Karaf tests.